src/components/bulkConcepts/component/PreviewCard.jsx

```diff
@@ -16,60 +35,78 @@ const PreviewCard = ({
     display_locale,
     id,
     url,
-    version,
-    created_on,
     concept_class,
     datatype,
     names,
   } = concept;
 
-  const DATE_OPTIONS = {
-    weekday: 'long',
-    year: 'numeric',
-    month: 'short',
-    day: 'numeric',
-    hour: 'numeric',
-    minute: 'numeric',
-  };
-
   const mapping = mappings || 'none';
   const description = descriptions ? descriptions[0].description : 'none';
-  const synonyms = names && names.map(name => name.name).join();
+  const synonyms = names && names
+    .map(name => `<span className='synonym'>${name.name}</span>`)
+    .filter(name => name !== `<span className='synonym'>${display_name}</span>`)
+    .join(' ');
+  let setMembers;
+  let answers;
+  let otherMappings = mapping;
+  let remainingCount;
+  if (mapping !== 'none') {
+    setMembers = getSetMembers(mapping);
+    answers = getAnswers(mapping);
+    otherMappings = getOtherMappings(mapping);
+  }
+
+  const displaySetMembersOrAnswers = () => {
+    if (setMembers && setMembers.length > 0) {
+      remainingCount = setMembers.length - 5;
+      return <CardBody
+        remainingCount={remainingCount}
+        title="Set members"
+        body={setMembers.slice(0, 5).join(' ')}
+        remainingItems={setMembers.slice(5).join(' ')}
+      />;
+    }
+    if (answers && answers.length > 0) {
+      remainingCount = answers.length - 5;
+      return <CardBody
+        remainingCount={remainingCount}
+        title="Answers"
+        body={answers.slice(0, 5).join(' ')}
+        remainingItems={answers.slice(5).join(' ')}
+      />;
+    }
+    return '';
+  };
+
   return (
     <div className="col-9">
       <Modal isOpen={open} className="modal-lg">
         <ModalBody className="preview-modal">
           <h6>
-            {id}
+            {`${display_name}(${id})`}
             <div className="card-version">
               <small>
-                <em>
-                  Version:&nbsp;
-                  {version}
-                </em>
-                <em>
-                  Created on:&nbsp;
-                  {new Date(created_on).toLocaleDateString('en-US', DATE_OPTIONS)}
-                </em>
+                <a href={`https://qa.openconceptlab.org${url}`} target="_blank" rel="noopener noreferrer">
```

Comment:
I see the above url scattered around in the code base. Shouldn't we have a constant for it?

Reply:
I have created a constant for it.